assets/styles/global.scss

```diff
@@ -215,23 +210,6 @@ hr {
   }
 }
 
-// Transak
-.transak_modal-overlay {
-  @apply z-[999] #{!important};
-}
-
-.transak_modal {
-  @apply z-[1000] max-h-[600px] #{!important};
-}
-
-#transakOnOffRampWidget {
-  height: 100% !important;
-}
```

Comment:
I think we used this class

<img width="888" alt="Capture d’écran 2024-02-07 à 2 01 26 PM" src="https://github.com/kodadot/nft-gallery/assets/9987732/8833eae6-40e2-4980-8b19-b69e28e69e0f">

Reply:
The 'Add Funds' modal doesn't open in production and in development this error is thrown: 
![Screenshot from 2024-02-08 13-29-31](https://github.com/kodadot/nft-gallery/assets/47194071/1608cd0d-07ff-4ebd-87a7-55f4bddaf925)